charts/fluentbit/values.yaml

```diff
@@ -0,0 +1,158 @@
+namespace:
+  create : true
+  name: "calyptia-fluentbit"
+
+serviceAccount:
+  create: true
+
+image:
+  amd:
+    repository: http://709825985650.dkr.ecr.us-east-1.amazonaws.com/calyptia/calyptia-fb
+    tag: 22.10.2-amd64
+    pullPolicy: Always
+  arn:
+    repository: nil
+    tag: nil
+  pullPolicy: Always
+
+arch: amd64
+
+metricsPort: 2020
+hostNetwork: false
+dnsPolicy: ClusterFirst
+livenessProbe:
+  httpGet:
+    path: /
+    port: http
+
+readinessProbe:
+  httpGet:
+    path: /api/v1/health
+    port: http
+extraVolumes: []
+securityContext: {}
+extraVolumeMounts: []
+
+existingConfigMap: ""
+
+
+service:
+  type: ClusterIP
+  port: 2020
+  annotations: {}
+
+rbac:
+  create: true
+  nodeAccess: false
+
+labels: {}
+annotations: {}
+podAnnotations: {}
+
+luaScripts: {}
+
+
+
+## https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/configuration-file
+config:
+  service: |
+    [SERVICE]
+        Daemon Off
+        Flush {{ .Values.flush }}
+        Log_Level {{ .Values.logLevel }}
+        Parsers_File parsers.conf
+        Parsers_File custom_parsers.conf
+        HTTP_Server On
+        HTTP_Listen 0.0.0.0
+        HTTP_Port {{ .Values.metricsPort }}
+        Health_Check On
+
+  ## https://docs.fluentbit.io/manual/pipeline/inputs
+  inputs: |
+    [INPUT]
+        Name tail
+        Path /var/log/containers/*.log
+        multiline.parser docker, cri
+        Tag kube.*
+        Mem_Buf_Limit 5MB
+        Skip_Long_Lines On
+
+    [INPUT]
+        Name systemd
+        Tag host.*
+        Systemd_Filter _SYSTEMD_UNIT=kubelet.service
+        Read_From_Tail On
+
+  ## https://docs.fluentbit.io/manual/pipeline/filters
+  filters: |
+    [FILTER]
+        Name kubernetes
+        Match kube.*
+        Merge_Log On
+        Keep_Log Off
+        K8S-Logging.Parser On
+        K8S-Logging.Exclude On
+
+  ## https://docs.fluentbit.io/manual/pipeline/outputs
+  outputs: |
+    [OUTPUT]
```

Comment:
Personally I detest the use of Elasticsearch as the default, it doesn't really help anyone.
Let's add `stdout` or `null`.
`stdout` is probably better but we must exclude our own logs via annotation.